package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "hemera-dynamodb-store",
+  "version": "1.0.0",
+  "description": "This is a plugin to use DynamoDb with Hemera",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./**/*.js --fix",
+    "test": "./node_modules/.bin/nyc ./node_modules/.bin/mocha test/*.js",
+    "coverage": "npm run test && ./node_modules/.bin/nyc report --reporter=html"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hemerajs/hemera-dynamodb-store.git"
+  },
+  "keywords": [
+    "dynamodb",
+    "hemerajs",
+    "dynamo",
+    "hemera",
+    "aws",
+    "db"
+  ],
+  "author": "Nikola Kusibojovski",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/hemerajs/hemera-dynamodb-store/issues"
+  },
+  "homepage": "https://github.com/hemerajs/hemera-dynamodb-store#readme",
+  "dependencies": {
+    "aws-sdk": "^2.173.0",
+    "hemera-plugin": "^0.1.0",
+    "hemera-store": "^1.1.15"
+  },
+  "devDependencies": {
+    "code": "^5.1.2",
+    "dynamodb-local": "0.0.18",
```

Comment:
This is no dev dependency

Reply:
You think on dynamodb-local, right ? So if it is not dev dependency, you think to put in dependencies ?